manual/de/ch_Finance-Quote.xml

```diff
@@ -14,14 +14,121 @@ Translators:
 -->
 <chapter id="finance-quote"
          xmlns:xi="http://www.w3.org/2001/XInclude">
-  <title>Die Online-Kursaktualisierung einrichten</title>
+  <title>Einrichtung der Kursaktualisierung</title>
 
-  <para>Wenn Sie mehr als ein paar Devisen und Wertpapiere besitzen, werden Sie es bald leid sein, die Kurse
-    ständig per Hand aktualisieren zu müssen. &app; bietet Ihnen die Möglichkeit, automatisch den
-    aktuellsten Kurs für Ihre Devisen und Wertpapiere über das Internet herunterzuladen. Hierfür
-    wird das &app-perl;-Modul &app-fq; verwendet.
+  <para>Wenn Sie mehr als ein paar Devisen und Wertpapiere besitzen, werden Sie es vielleicht bald leid
+    sein, die Kurse ständig per Hand aktualisieren zu müssen. &app; bietet Ihnen die Möglichkeit,
+    automatisch den aktuellsten Kurs für Ihre Devisen und Wertpapiere über das Internet
+    herunterzuladen. Hierfür wird das &app-perl;-Modul &app-fq; verwendet.
   </para>
 
+  <itemizedlist>
+    <title>Die Vorraussetzungen für eine Online-Kursabfrage mit &app-fq; sind</title>
+
+    <listitem>
+      <para>ein &app-perl; Interpreter und
+      </para>
+    </listitem>
+
+    <listitem>
+      <para>das &app-perl;-Module &app-fq; mit allen Abhängigkeiten.
+      </para>
+    </listitem>
+  </itemizedlist>
+
+  <note>
+    <para>Wenn Sie sich bei der Durchführung mit den nachfolgend beschriebenen Schritte nicht sicher fühlen,
```

Comment:
Mit ein paar Schritten Abstand habe ich gedacht: Müssen wir das in jedem Kapitel extra erwähnen? Dafür haben wir doch Getting-Help.

Vielleicht eher am Schluß: Wenn etwas nicht geklappt hat, —mit ausfühlicher Beschreibung —Betriebssystem/Distribution, GnuCash-Version, durchgeführte Schritte—…
Der Einschub gehört aber auch eher direkt in getting-help.

Reply:
Du hast recht, ich formuliere hier das um und setze einen Link auf getting-help.

Die Position vorne in der Einleitung halte ich für gut; oft fängt der Anwender vorne zu lesen an und liest selten bis zu Ende